tools/anthosbm-ansible-module/roles/ws-docker/tasks/main.yml

```diff
@@ -13,26 +13,15 @@
 # limitations under the License.
 
 ---
-- name: Remove previous docker versions [Ubuntu]
```

Comment:
The reason for removing the packages before installation is to keep the role idempotent. Please keep this task.